### ISSUE TITLE
warnings for unverified https request spamming the log - can this be turned off? 

### ISSUE THREAD

**Issue by jata1** [travisghansen/hass-opnsense#177]
Hi guys,

Thanks for the work you have been doing to get this integration working with 24.7. 

I have a problem using my duckdns acme certificate as I think the integration does not like my setup with adguard DNS and/or my reverse proxy.

So my setup is using https://ip_opnsense (without ssl certificate) and is working well but the log is spammed every time there is an api call - so literally 1000s of times per day.

Is there any way you can add something to disable the logging of this warning please? - see below.

```
2024-09-10 07:48:03.972 WARNING (SyncWorker_54) [py.warnings] /usr/local/lib/python3.12/site-packages/urllib3/connectionpool.py:1064: InsecureRequestWarning: Unverified HTTPS request is being made to host '192.168.1.1'. Adding certificate verification is strongly advised. See: [https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings](https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings%3C/span%3E)
  warnings.warn(
```

**Comment by Snuffy2** [comment]
It has already been fixed and will be available in the next release.  https://github.com/travisghansen/hass-opnsense/commit/bf692d61bd0051869cac426a4ca977a9988ab0d0

**Comment by jata1** [comment]
That's great. Thanks again.

**Comment by alexdelprete** [comment]
PR #164 was merged and is in in [v0.2.1-beta.1](https://github.com/travisghansen/hass-opnsense/releases/tag/v0.2.1-beta.1), available since yesterday. You can try it. :)
